auditorium/views/decorators/with-layout.js

```diff
@@ -8,14 +8,37 @@ function withLayout (headline) {
       var flash = state.flash
       state.flash = null
       return html`
-        <div data-role="app-host" class="mw8 center pa3 pb5 f5 roboto dark-gray">
-          <div id="headline">
-            <h1 class="f2 normal mt0 mb4">${headline || __('Offen Auditorium')}</h1>
-            ${flash ? html`
-              <p data-role="flash-message" class="dib pa2 br2 bg-black-05 mt0 mb2">${flash}</p>
-            ` : null}
+        <div data-role="app-host" class="f5 roboto dark-gray">
+          <div class="w-100 h3 bg-black-05">
+            <div class="mw8 center flex ph3 pt2" id="headline">
+              <img src="/offen-icon-black.svg" alt="Offen logo" width="37" height="40" class="ma0 mt1 mr3">
+              <h1 class="f2 normal ma0 mt1">${headline || __('Offen Auditorium')}</h1>
+            </div>
+          </div>
+          <div class="mw8 center ph0 ph3-ns pb4">
+            <div>
+              ${flash ? html`
+                <p data-role="flash-message" class="dib pa2 br2 ma0 mt3 ml3 ml0-ns mr3 mr0-ns bg-light-yellow">${flash}</p>
+              ` : null}
+            </div>
+            ${originalView(state, emit)}
+          </div>
+          <div class="mw8 center flex flex-column flex-row-ns justify-between ph3 pb5 moon-gray">
+            <div>
+              <p class="b ma0 mb1">
+                Offen
+              </p>
+              <p class="ma0 mb1">
+               ${__('Transparent web analytics')}
```

Comment:
Breaking the text into two parts like this will be kind of awkward when this gets translated. Would it be an option to use a `<br>` and then use the line-height of the paragraph for spacing:

```html
<p class="ma0 mb2"> <!-- add line-height here if needed -->
  ${raw(__('Transparent web analytics<br>for everyone'))}
</p>
```